plugins/get_url/komand_get_url/util/utils.py

```diff
@@ -79,13 +79,18 @@ def check_prefix_and_download(
         if insightconnect_plugin_runtime.helper.check_cachefile(meta.get("metafile")):
             headers = self.check_url_meta_file(meta)
 
+        # Caching headers from meta info can be None
+        # request.urlopen expects str or bytes
+        etag = headers.get("etag", "") or ""
+        last_modified = headers.get("last-modified", "") or ""
+
```

Comment:
Fix here to explicitly set these header values to `""` when they are `null` in the meta cache file

Otherwise `requests.urlopen` returns an exception